readthedocs/oauth/management/commands/reconnect_remoterepositories.py

```diff
@@ -56,20 +56,19 @@ def _connect_repositories(self, organization, no_dry_run, only_owners):
             Q(clone_url__in=Subquery(organization.projects.values('repo')))
         )
         for remote in RemoteRepository.objects.filter(remote_query).order_by('created'):
-            admin = json.loads(remote.json).get('permissions', {}).get('admin')
+            admin = remote.remote_repository_relations.filter(
+                user__owner_organizations=organization,
+                admin=True
+            ).exists()
 
-            if only_owners and remote.users.first() not in organization.owners.all():
```

Comment:
I'm not sure what to do here and if it's required anymore or not, I'm not familiar with Organization code :/